targets/circuitplay-bluefruit.json

```diff
@@ -0,0 +1,9 @@
+{
+    "inherits": ["nrf52840"],
+    "build-tags": ["circuitplay_bluefruit"],
+    "flash-method": "msd",
+    "msd-volume-name": "CPLAYBTBOOT",
+    "msd-firmware-name": "firmware.uf2",
+    "uf2-family-id": "0xADA52840",
```

Comment:
I was only able to make this work by specifying the uf2-family-id value as a base 10 integer, ie:
```
"uf2-family-id": 2913282112,
```
otherwise `encoding/json` complains about attempting to unmarshal a string into a uint32

Reply:
Done